recipe/meta.yaml

```diff
@@ -1,27 +1,34 @@
 {% set name = "ruptures" %}
-{% set version = "1.0.6" %}
+{% set version = "1.1.2rc1" %}
```

Comment:
Don't publish release candidates on the main label.

Reply:
Noted. Then I'll wait a few days to push the 1.1.2 version. In the meantime I have two questions:

- can you merge #8 in which I added myself and @oboulant as maintainers?
- can you point to some documentation for pushing release candidates? I have seen some discussions but nothing "official".

Reply:
Have a look at https://github.com/conda-forge/cfep/blob/master/cfep-05.md

Reply:
Many thanks!